resources/com.sap.piper/pipeline/stashSettings.yml

```diff
@@ -4,6 +4,8 @@ Init:
     - name: "source"
       includes: "**/*"
       excludes: ".pipeline/**"
+    - name: "application"
+      includes: "**/*"
```

Comment:
Are you scared that the user hides files in the .pipeline dir?

Reply:
I responded to your Teams message